docs/api_static/plasmapy.conftest.rst

```diff
@@ -0,0 +1,10 @@
+:orphan:
+
+`plasmapy.conftest`
+===================
+
+.. currentmodule:: plasmapy.conftest
+
+.. automodapi::  plasmapy.conftest
+   :include-all-objects:
+   :no-heading:
```

Comment:
I was surprised to see what I see as essentially a pytest config file documented. But sure, there's no harm in putting it in here.

Reply:
Yeah, I debated this, along with documenting private members.  I deferred the decision on the later due to `automodapi` not having convenient mechanisms for this.  I included the module because I thought it might be useful for contributors down the road.